interface/src/app/analyze/analysis/analysis.js

```diff
@@ -124,21 +131,11 @@ AnnotationType, SampleBin) {
       }
     ],
 
-    'axes': [
-      // this generated x-axis is useless, so we make one in marks below
-      // {"type": "x", "scale": "nodes", "title": "Node ID"},
-      {
-        'type': 'y', 'scale': 'samples', 'title': 'sample'
-        // FIXME vega doesn't like either of these attempts at labels
-        // "properties": {
-        //   "labels": {
-        //     // "text": {"field": "sample_object.ml_data_source"}
-        //     "text": {"template": "test
-        // {{datum.data.sample_object.ml_data_source}}"}
-        //   }
-        // }
-      }
-    ],
+    // 'axes': [
```

Comment:
Are we not going to use this 'axes' value (because of the new method of building the axes labels)? Will we ever need these lines again?